api/fittings/lisk_params_validator.js

```diff
@@ -39,7 +39,13 @@ module.exports = function create (fittingDef, bagpipes) {
 
 				error.errors = _.map(validateResult.errors, function (e) {
 					var errors = _.pick(e, ['code', 'message', 'in', 'name', 'errors']);
-					errors.errors = _.map(e.errors, function (e) { return _.pick(e, ['code', 'message'] ); });
+					errors.errors = _.map(e.errors, function (e) {
+						var paramError = _.pick(e, ['code', 'message','path'] );
+						if (_.isArray(paramError.path)) {
+							paramError.path = paramError.path.join(' > ');
+						}
+						return paramError;
```

Comment:
We should send `path` as an array without concatenating. So end user can utilize this information as per his UI and implementation.